src/components/entities/products/ProductPage.vue

```diff
@@ -116,10 +151,46 @@ const Component = defineComponent({
       rules,
       isNew,
       listCategory,
+      previewImageMaxSize,
     };
   },
+
+  methods: {
+    openUploadDialog() {
+      // console.log(input.value)
+      // input.value.click()
+      const fileInput: HTMLInputElement | any = document.getElementsByClassName('hidden-input')[0]
+      fileInput && fileInput.click()
+      console.log(fileInput)
+    },
+    uploadImage(event: any) {
+      const file = event.target.files?.[0];
+      if (!file) return;
+      if (file.size > this.previewImageMaxSize) {
+        dispatchAlert(AlertColor.ERROR, 'File is too big');
```

Comment:
Weird, you've an error message but it's not visible on your demo.

Reply:
Most likely because alerts are used from via the store and not used inside the components directly, i.e. component is an isolated thing and knows nothing about the outer world much.